.github/workflows/build-node-binaries.yaml

```diff
@@ -0,0 +1,45 @@
+# This defines a workflow to build and release a new version of the aptos CLI.
+# In order to trigger it go to the Actions Tab of the Repo, click "Release CLI" and then "Run Workflow".
+
+name: "Build Aptos Node Binaries"
+
+on:
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      git_ref:
+        type: string
+        required: true
+        description: "The ref to build from i.e. aptos-node-vX.X.X"
+
+jobs:
+  build-node-binary:
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, ubuntu-22.04]
+    name: "Build Aptos Node Binary on ${{ matrix.os }}"
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # pin@v3
+        with:
+          # ref: ${{ aptos-node-v1.4.3 }}
+          ref: aptos-node-v1.4.3
```

Comment:
aptos-node-* branches are deprecated in favor of aptos-release-*

Reply:
1) This is for debugging, it will actually be a variable
2) This is a tag, the branches are called aptos-release but the tags are aptos-node / aptos-framework etc